here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/util/RequestTypesUtil.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2017-2024 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+package com.here.naksha.lib.handlers.util;
+
+import naksha.model.Naksha;
+import naksha.model.request.Request;
+import naksha.model.request.Write;
+import naksha.model.request.WriteRequest;
+import org.jetbrains.annotations.NotNull;
+
+public final class RequestTypesUtil {
```

Comment:
Is it tested? I see two static methods - should be a quick one ;)

Reply:
added